internal/controlplane/handlers_githubwebhooks.go

```diff
@@ -641,33 +624,32 @@ func (s *Server) parseArtifactPublishedEvent(
 	return eiw.ToMessage(msg)
 }
 
-func parsePullRequestModEvent(
+func (s *Server) parsePullRequestModEvent(
 	ctx context.Context,
 	whPayload map[string]any,
 	msg *message.Message,
 	dbrepo db.Repository,
-	store db.Store,
-	prov *providers.ProviderBuilder,
+	providerID uuid.UUID,
 	action string,
 ) error {
-	// NOTE(jaosorior): this webhook is very specific to github
```

Comment:
This comment is still true, we just think it's obvious, correct?

Reply:
Correct - but this particular codepath is so Github-specific that it's not really worth noting any more.